sky/client/sdk.py

```diff
@@ -716,7 +716,7 @@ def exec(  # pylint: disable=redefined-builtin
 @usage_lib.entrypoint
 @server_common.check_server_healthy_or_start
 @annotations.client_api
-@rest.retry_on_server_unavailable()
+@rest.retry_transient_errors()
```

Comment:
If we reconnect, there is no guarantee that the logs will be perfectly consistent, right? It seems like we will possibly skip some new logs that were emitted during the disconnection.
If this is true, I think we should at least print some warning to avoid significant user confusion.

Reply:
We maintain the line processed in [`RetryContext`](https://github.com/skypilot-org/skypilot/blob/97aa3b08173c7d1f284f08ce6bd45d26ee039d30/sky/server/rest.py#L42) and skips the processed lines during retry. So the log is consistent as long as the log tailing logic on server does not change between restarts, which should be an separate issue. Wdyt? @cg505

Reply:
Great, missed this.